mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -220,10 +220,10 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		FIELD field_23186 emissiveLightingPredicate Lnet/minecraft/class_4970$class_4973;
 		METHOD <init> (Lnet/minecraft/class_3614;Ljava/util/function/Function;)V
 			ARG 1 material
-			ARG 2 materialColorFactory
+			ARG 2 mapColorFactory
```

Comment:
What makes this a factory? Is it constructing new colors? If not, it is a provider.

Reply:
Might have been matcher, since one of the constructors is a `Function<BlockState, MapColor>`